crates/shared/src/sources/balancer_v2/pools/common.rs

```diff
@@ -224,20 +279,64 @@ fn scaling_exponent_from_decimals(decimals: u8) -> Result<u8> {
         .ok_or_else(|| anyhow!("unsupported token with more than 18 decimals"))
 }
 
+/// An internal utility method for sharing the success value for an
```

Comment:
Isn't this code already part of a previous PR?

Reply:
It just moved here.